evap/contributor/tests/test_views.py

```diff
@@ -86,21 +96,23 @@ class TestContributorEvaluationView(WebTestWith200Check):
 
     @classmethod
     def setUpTestData(cls):
-        create_evaluation_with_responsible_and_editor(evaluation_id=TESTING_COURSE_ID)
+        result = create_evaluation_with_responsible_and_editor(evaluation_id=TESTING_COURSE_ID)
+        cls.responsible = result['responsible']
+        cls.editor = result['editor']
```

Comment:
`WebTestWith200Check` still works with the email addresses. I would remove `test_users` as class attribute and create it here in `setUpTestData`, like:
```python
cls.test_users = [cls.responsible, cls.editor]
```
I think this makes it clearer which users get tested by the simple 200 check